### ISSUE TITLE
singleSelection: true; prevents use of <c-badge>

### ISSUE THREAD

**Issue by andrzejkala** [CuppaLabs/angular2-multiselect-dropdown#169]
I've noticed that setting the option `singleSelection: true` somehow prevents the use of  `<c-badge>` on selected item. 

When I remove the option (or set it to `false`) it works as intended (but I do need the option to only select one.

The code in question:
```html
<angular2-multiselect
    [data]='package.statuses'
    [(ngModel)]='package.selectedStatuses'
    [settings]='{ singleSelection: true, canenableCheckAll: false, classes: "pp-multiselect-radio pp-multiselect", enableSearchFilter: true }'
    (onSelect)="changeStatus(package)"
    (onDeSelect)="changeStatus(package)"
    (onSelectAll)="changeStatus(package)"
    (onDeSelectAll)="changeStatus(package)">

    <c-item>
        <ng-template let-item="item">
            <span class="status-item">
               <img class="status-icon" popper="{{item.name}}" src="/assets/icons/packageStatus/{{item.code}}.svg" />
                <span class="status-desc">{{item.name}}</span>
            </span>
         </ng-template>
    </c-item>

    <c-badge>
        <ng-template let-item="item">
         <span class="status-item">
         <img class="status-icon" popper="{{item.name}}" src="/assets/icons/packageStatus/{{item.code}}.svg" />
         <span class="status-desc">{{item.name}}</span>
         </span>
        </ng-template>
    </c-badge>
</angular2-multiselect>
```

**Comment by ikramcheb** [comment]
Hi, Is there any news on this issue?

**Comment by Cobr3n** [comment]
Hi man can u solve this problem ?

**Comment by taufeek6** [comment]
Untill they solve it,   <c-badge *ngIf="!dropdownSettings.singleSelection"> could be a solution.

**Comment by CuppaLabs** [comment]
fixed. Fix available in v2.10.2
